src/test/bridges/liquity/StakingBridgeUnit.t.sol

```diff
@@ -103,7 +99,7 @@ contract StakingBridgeUnitTest is TestUtil {
         );
 
         while (i < numIters) {
-            depositAmount = rand(depositAmount);
+            depositAmount = _rand(depositAmount);
```

Comment:
You can have a list as inputs to have the fuzzer find values instead of trying to do random values inside the solidity.

Reply:
I wrote the Liquity tests when I didn't know foundry yet which is why I didn't you fuzzying back then.